docs/modules/deploy/partials/nav.adoc

```diff
@@ -10,6 +10,7 @@
 ** xref:deploy:deploying-in-cloud.adoc[]
 ** xref:deploy:installing-using-docker.adoc[]
 ** xref:deploy:deploying-in-kubernetes.adoc[]
+** xref:deploy:deploying-with-operator.adoc[]
```

Comment:
This should probably be a third-level item since it relates to deploying on Kubernetes.
```suggestion
*** xref:deploy:deploying-with-operator.adoc[]
```